src/core/createEventManager.js

```diff
@@ -27,7 +27,7 @@ export default ({ createEvent, optIn, lifecycle, network, config, logger }) => {
   const addMetaTo = payload => {
     const meta = {
       gateway: {
-        imsOrgId
+        orgId
```

Comment:
I think this is bad, because Konductor is still expecting `imsOrgId`.

Reply:
@Aaronius We created a ticket for them to rename it and reached out to Rares. We will wait until Konductor has published before we merge this PR.

Reply:
Okay, cool. Thanks.

Reply:
I just saw "Please don't merge until the change is made on Konductor." in your description. I'm sorry for not reading the description. 😞